packages/admin-panel-server/src/connections/MeditrakConnection.ts

```diff
@@ -2,14 +2,30 @@
  * Tupaia
  * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
-
-import { QueryParameters, ApiConnection } from '@tupaia/server-boilerplate';
+import camelcaseKeys from 'camelcase-keys';
+import { AccessPolicy } from '@tupaia/access-policy';
+import { QueryParameters } from '@tupaia/server-boilerplate';
+import { SessionHandlingApiConnection } from './SessionHandlingApiConnection';
+import { BES_ADMIN_PERMISSION_GROUP } from '../constants';
 
 const { MEDITRAK_API_URL = 'http://localhost:8090/v2' } = process.env;
 
-export class MeditrakConnection extends ApiConnection {
+const isBESAdmin = (policy: Record<string, string[]>) => {
+  return new AccessPolicy(policy).allowsSome(null, BES_ADMIN_PERMISSION_GROUP);
+};
```

Comment:
I was thinking maybe we can have a middleware that gets run for all the viz-builder-route that can check for permission. However I haven't got that done yet, so I think for now this should be ok!

Reply:
Yes that's a good idea. Yes sounds like it will just be easiest for you to refactor it when you get to it so I'll just leave it as is.